lib/jnpr/junos/utils/sw.py

```diff
@@ -59,9 +59,10 @@ class SW(Util):
     def __init__(self, dev):
         Util.__init__(self, dev)
         self._dev = dev
-        self._RE_list = [
-            x for x in dev.facts.keys() if x.startswith('version_RE')]
-        self._multi_RE = bool(len(self._RE_list) > 1)
+        self._RE_list = []
+        if 'junos_info' in dev.facts and dev.facts['junos_info'] is not None:
```

Comment:
Is this code backward compatible? I think we dont have 'junos_info' earlier hence in absence of else self._RE_list will be a empty list only.
Please check